### DISCUSSION TITLE
Range input field

### DISCUSSION THREAD

**Discussion by ryanleichty** [craftcms/cms#15804] [Ideas] [answered]
Is it possible to get a native range (slider) input field? Or perhaps add a style option to the number field (with the `step` option)?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range

**Comment by ryanleichty** [comment]
This is a common input type in Shopify

<img width="320" src="https://github.com/user-attachments/assets/76e4b7d9-86b0-4fde-8b69-56a4d14ec2ad">

<img width="320" src="https://github.com/user-attachments/assets/753ef948-ea9a-4cd1-aaed-6f0b36fcdae1">

<img width="320" src="https://github.com/user-attachments/assets/146ce077-300b-44b4-8bdc-e9389a76af22">

**Comment by brandonkelly** [comment, marked as answer]
Added for Craft 5.5! (#15972)

**Reply by ryanleichty** [reply]
Awesome, thank you! 🔥

**Reply by brandonkelly** [reply]
Craft 5.5.0 is out with the new Range field type 🎉
